tests/manual-test-cases/Group5-Functional-Tests/5-25-OPS-User-Grant.robot

```diff
@@ -74,7 +74,11 @@ vic-machine create grants ops-user perms
     Should Be Equal As Integers  ${rc}  0
     ${rc}  ${c3}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --net public -v barVol:/dir ${busybox}
     Should Be Equal As Integers  ${rc}  0
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} rm -f ${c1} ${c2} ${c3}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create newvol
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${c4}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d -v newvol:/dir ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} rm -f ${c1} ${c2} ${c3} ${c4}
```

Comment:
Can we have a `docker volume rm` as well.

Reply:
Added and verified that the test passes in DRS and non-DRS envs.